docs/coding.rst

```diff
@@ -360,3 +360,432 @@ Bedrock includes an `.editorconfig` file in the root directory that you can
+
+    Default: False
+
+    Example: ``include_highres_image=True``
+
+- l10n_image
+    Boolean to indicate if image has translatable text.
+
+    Default: False
+
+    Example: ``l10n_image=True``
+
+
+- lazy_loading
+    Boolean to provide “lazy” value for “loading” attribute. This will be “eager” by default. Lazy loading defers fetching of images to a browser decision based on user scroll and connection.
+
+    Default: False
+
+    Example: ``lazy_loading=True``
+
+
+
+Call out
+^^^^^^^^
+
+- title
+    **Required**. String indicating heading text (usually a translation id wrapped in ftl function).
+
+    Default: N/A
+
+    Example: ``title=ftl('firefox-privacy-hub-read-the-privacy-notice-for')``
+
+- desc
+    String indicating paragraph text (usually a translation id wrapped in ftl function).
+
+    Default: None 
+
+    Example: ``desc=ftl('firefox-channel-test-beta-versions-of-firefox-ios')``
+
+- class
+    String adding class(es) to the section tag.
+
+    Default: None 
+
+    Example: ``class='mzp-t-firefox ' + product_class``   
+
+- include_cta 
+    Boolean indicating whether or not to include the body of the macro call (usually a mix of text and html).
+
+    Default: None 
+
+    Example: ``include_cta=True`` 
+
+- heading_level
+    Number indicating heading level for title text. Should be based on semantic meaning, not presentational styling.
+
+    Default: 2
+
+    Example: ``heading_level=1``            
+
+
+Split
+^^^^^
+
+- block_id
+    String providing id to the section tag (usually if it needs to be used as an in-page link).
+
+    Default: None 
+
+    Example: ``id='nextgen``
+
+- base_el
+    String for block HTML tag not required.
+
+    Default: section
+
+    Example: ``base_el='aside'``
+
+- block_class
+    String providing class(es) to the section tag.
+
+    Default: None 
+
+    Example: ``block_class='mzp-l-split-reversed mzp-l-split-center-on-sm-md``
+
+- theme_class
+    String providing theme class(es) to a container div tag inside the section.
+
+    Default: None 
+
+    Example: ``theme_class='mzp-t-dark'``   
+
+- body_class
+    String providing class(es) to the body (text content) div inside the section.
+
+    Default: None 
+
+    Example: ``Not currently in use`` 
+
+- image_url
+    Path to image location.
+
+    Default: None 
+
+    Example: ``image_url=’img/firefox/accounts/trailhead/value-respect.jpg’`` 
+
+- media_class
+    String providing class(es) to the media div inside the section.
+
+    Default: None 
+
+    Example: ``media_class='mzp-l-split-h-center'``
+
+- include_highres_image
+    Boolean that determines whether the image can also appear in high res.
+
+    Default: False
+
+    Example: ``include_highres_image=True``   
+
+- l10n_image 
+    Boolean to indicate if image has translatable text.
+
+    Default: False
+
+    Example: ``l10n_image=True`` 
+
+- image_alt
+    String providing alt text to the image.
+
+    Default: `'
```

Comment:
Since `image_alt` would be a string, its default value would usually be an empty string, best represented by an opening and closing quote (i.e. '' or ‘’  (single opening and closing quotes) )